### PR TITLE
Add error to message

### PR DESCRIPTION
**Pull request by yaacov**
**Description**

We currently have a TODO handle error messages in the stomp implementation, this PR implements an error handling mechanism for error messages.

### CODE REVIEW

**Comment by yaacov** [comment]
@jhernand merging without reviews because this is a trivial change we can revert later, and I do not want to get conflicts with other PR's.

File: cmd/messaging-tool/receive.go [hunk 1]
```diff
@@ -31,13 +31,21 @@ var receiveCmd = &cobra.Command{
...
 
-func callback(m client.Message, destination string) (err error) {
+func callback(message client.Message, destination string) (err error) {
+	if message.Err != nil {
+		glog.Errorf(
+			"Can't subscribe to destination '%s': %s",
+			destinationName,
+			message.Err.Error(),
+		)
+		return
```
Comment (cben) [inline cmd/messaging-tool/receive.go]: The callback has return type `error` - should this maybe return an error obj?

Comment (yaacov) [reply cmd/messaging-tool/receive.go]: Currect !!! Thanks ! fixing now !

File: pkg/connections/stomp/subscribe.go [hunk 2]
```diff
@@ -26,18 +26,16 @@ func (c *Connection) Subscribe(destination string, callback func(m client.Messag
...
 
 	// Wait for messages:
 	for message := range subscription.C {
-		if message.Err != nil {
-			// TODO: error ?
-			break
-		}
-
-		callback(client.Message{Body: string(message.Body)}, destination)
+		callback(
```
Comment (cben) [inline pkg/connections/stomp/subscribe.go]: ... though we don't do anything with callback's return value.

**Pull request merged**